editor/modes/visual-editor/style.scss

```diff
@@ -255,3 +255,39 @@ $sticky-bottom-offset: 20px;
 		content: '';
 	}
 }
+
+.editor-visual-editor__placeholder {
```

Comment:
It's hard to extract something meaningful really. I don't like extracting shared styles just because they are the same. My questions here are:

 - How should we name the extracted shared CSS mixin?
 - Is it really worth it if it's shared between two elements only